examples/aws/terraform/starter-cluster/Makefile

```diff
@@ -1,37 +1,39 @@
+#TF_LOG=DEBUG
```

Comment:
Can you remove those changes from the PR?

Reply:
yes, will do. sorry about that.